### ISSUE TITLE
python: symbol lookup error

### ISSUE THREAD

**Issue by Mickeybaby** [layumi/Person_reID_baseline_pytorch#139]
(reID) mickey@a208-System-Product-Name:~/vision/Person_reID_baseline_pytorch$ python train.py --gpu_ids 0 --name ft_ResNet50 --train_all --batchsize 32  --data_dir /home/mickey/vision/Market/pytorch/
This is not an error. If you want to use low precision, i.e., fp16, please install the apex with cuda support (https://github.com/NVIDIA/apex) and update pytorch to 1.0
[Resize(size=(256, 128), interpolation=PIL.Image.BICUBIC), Pad(padding=10, fill=0, padding_mode=constant), RandomCrop(size=(256, 128), padding=None), RandomHorizontalFlip(p=0.5), ToTensor(), Normalize(mean=[0.485, 0.456, 0.406], std=[0.229, 0.224, 0.225])]
python: symbol lookup error: /home/mickey/miniconda3/envs/reID/lib/python3.6/site-packages/torch/lib/libtorch_python.so: undefined symbol: PySlice_Unpack

**Comment by layumi** [comment]
Hi @Mickeybaby 
Sorry for the late response.
It seems to be an error with pytorch. You may check this issue.
https://github.com/pytorch/pytorch/issues/14931

You need to upgrade your python to 3.6.x

**Comment by Mickeybaby** [comment]
Thanks.
I solved it.
Best wishes!
